Dockerfile

```diff
@@ -1,26 +1,43 @@
 FROM python:3.7-slim-buster
 
 ARG connectors=all
-COPY . /app
 
 RUN apt-get -qq update \
     && apt-get -qqy --no-install-recommends install \
         apt-utils \
         alien \
+        gnupg \
         libaio1 \
-        mongo-tools \
         mbuffer \
         wget \
     && rm -rf /var/lib/apt/lists/* \
-    && pip install -U --no-cache-dir pip \
+    && pip install -U --no-cache-dir pip
+
+# Add Mongodb ppa
+RUN wget -qO - https://www.mongodb.org/static/pgp/server-4.4.asc | apt-key add - \
```

Comment:
Can you combine this `RUN` with the one above?

Reply:
Yes, but it will not reduce a number of `apt-get update/install` commands because `slim` image does not have `wget` and `gnupg` pre-installed.

Reply:
Given that it seemed to me cleaner to separate them into different cmds

Reply:
Ah right!